pkg/apis/kubermatic/v1/policy_binding.go

```diff
@@ -71,9 +71,6 @@ type PolicyBindingSpec struct {
 	// PolicyTemplateRef references the PolicyTemplate by name
 	PolicyTemplateRef corev1.ObjectReference `json:"policyTemplateRef"`
 
-	// NamespacedPolicy is a boolean to indicate if the policy binding is namespaced
-	NamespacedPolicy bool `json:"namespacedPolicy,omitempty"`
-
```

Comment:
Why are we removing this?

Reply:
I would assume that we planned (since the sig cluster meeting) to embed the entire Kyverno object here, so we could deduce this information from the Kyverno object's kind (ClusterPolicy vs. Policy).

I don't really have strong opinions regarding this field. Do we "need" it in the KKP UI?

Reply:
Anyway, I can't really spend more time on this topic. So from my side, it's fine with whatever you guys agree upon and implement. This PR practically blocks https://github.com/kubermatic/dashboard/pull/7106 and Ahmad would have to adjust API and UI(WIP) work from his side according to the final output of the PR.

So I'll leave it between you guys to figure it out :shipit: cc @ahmadhamzh